src/core/tools/common/simple_logger.cpp

```diff
@@ -199,10 +199,10 @@ namespace dsn {
             if (_index - _start_index > 20)
             {
                 std::stringstream str2;
-                str2 << "log." << (_start_index + 1) << ".txt";
-                if (!::remove(str2.str().c_str()))
+                str2 << "log." << _start_index << ".txt";
```

Comment:
my bad - there are similar cases in other loggers. It is great if you can fix them together~